### PR TITLE
Make global tool

### PR DESCRIPTION
**Pull request by terrajobst**
This makes the tool a .NET Core global tool. I suggest you also push it to NuGet, this way it's easier to use. The name is specific enough that we shouldn't need to worry about naming conflicts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
+To run the tool just run: `apply-nullable-decorators [action] [args]` under the
+`src` directory.
 
 ### apply
 
-Loads the project, finds the type, and based on its analysis, rewrites the files with `?` annotations.
+Loads the project, finds the type, and based on its analysis, rewrites the files
+with `?` annotations.
+
+    usgage: apply-nullable-decorators apply <project> <type> [-msbuild value]
```
Comment (safern) [inline README.md]: usgage => usage

```suggestion
    usage: apply-nullable-decorators apply <project> <type> [-msbuild value]
```

File: README.md [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+    <project>           CS Project containing type to annotate (string, required)
+    <type>              Full type name to apply decorators to (string, required)
+    -msbuild <value>    Full path of MSBuild instance to use to load the
+                        workspace (string, default=)
 
 ### apistats
 
 Prints a summary of total number of public and internal/private APIs that are in the project.
+
+    usgage: apply-nullable-decorators apply <project> <type> [-msbuild value]
```
Comment (safern) [inline README.md]: Ditto

```suggestion
    usage: apply-nullable-decorators apply <project> <type> [-msbuild value]
```

File: README.md (hunk 1 above)
Comment (terrajobst) [reply README.md]: Hah, you passed the test ;-)

File: build.cmd [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@echo off
+setlocal
+
+set SLN=%~dp0src\apply-nullable-decorators.csproj
```
Comment (safern) [inline build.cmd]: There is actually a SLN file, do we want to build the solution itself?

**Review by safern — APPROVED** [review]
Nice stuff! Thanks for adding this.

File: build.cmd (hunk 3 above)
Comment (terrajobst) [reply build.cmd]: Good point, will fix

**Pull request merged**